src/api/index.test.js

```diff
@@ -581,34 +696,228 @@ it('getNamespaces', () => {
     items: 'namespaces'
   };
   fetchMock.get(/namespaces/, data);
-  return getNamespaces().then(response => {
+  return index.getNamespaces().then(response => {
     expect(response).toEqual(data.items);
     fetchMock.restore();
   });
 });
 
+it('getServiceAccount', () => {
+  const data = { items: 'serviceaccounts' };
+  fetchMock.get(/serviceaccounts/, data);
+  return index.getServiceAccount('default', 'default').then(response => {
+    expect(response).toEqual(data);
+    fetchMock.restore();
+  });
+});
+
 it('getServiceAccounts', () => {
   const data = { items: 'serviceaccounts' };
   fetchMock.get(/serviceaccounts/, data);
-  return getServiceAccounts().then(response => {
+  return index.getServiceAccounts().then(response => {
     expect(response).toEqual(data.items);
     fetchMock.restore();
   });
 });
 
+it('getSecretServiceAccountList No Secrets Matched', () => {
```

Comment:
I see the same test name later...

Reply:
First one relates to if a secret isn't patched to a service account and checks it doesn't fall over and break if its not patched to a service account
Changed the name of the test now